data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -136,6 +136,17 @@ local raid(level='raid0', uefi=false) = {
         mdswap_partition,
       ],
     },
+  ] else if boot_type == 'prep' then [
+    // PowerPC PReP boot partition case
+    {
+      search: '*',
+      partitions: [
+        { delete: true, search: '*' },
```

Comment:
why are you deleting partitions if we have fresh disk in qemu for the 3 architectures? shouldn't be needed.